src/modules/update.js

```diff
@@ -0,0 +1,57 @@
+export default class Task {
+  constructor(description, index) {
+    this.description = description;
+    this.completed = false;
+    this.index = index;
+  }
+
+  static task() {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      tasks = [];
+    } else {
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+    }
+    return tasks;
+  }
+
+  static addTask(task) {
+    const taskItem = document.querySelector('textarea');
+    const tasks = this.task();
+    tasks.push(task);
+    taskItem.value = '';
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+    window.location.reload();
+  }
+
+  static showTasks(task) {
+    const taskList = document.getElementById('task-list');
+
+    const div = document.createElement('div');
+    div.setAttribute('draggable', 'true');
+    div.setAttribute('class', 'listed-task');
+    div.setAttribute('id', `${task.index}`);
+    div.setAttribute('completed', `${task.completed}`);
+
+    const check = document.createElement('input');
+    check.setAttribute('type', 'checkbox');
+    check.setAttribute('class', 'check');
```

Comment:
- As the above part, this could also use a single input attribute value to ensure you avoid any repetition.